embedding/csharp/Tizen.Flutter.Embedding/ElmFlutterView.cs

```diff
@@ -0,0 +1,189 @@
+        private int InitialWidth { get; set; } = 0;
+
+        /// <summary>
+        /// The initial height of the view. Defaults to the parent height if the value is zero.
+        /// </summary>
+        private int InitialHeight { get; set; } = 0;
+
+        /// <summary>
+        /// Whether the view is running.
+        /// </summary>
+        public bool IsRunning => !View.IsInvalid;
+
+        /// <summary>
+        /// The current width of the view.
+        /// </summary>
+        public int Width
+        {
+            get
+            {
+                Debug.Assert(IsRunning);
+
+                return EvasObject.Geometry.Width;
+            }
+        }
+
+        /// <summary>
+        /// The current height of the view.
+        /// </summary>
+        public int Height
+        {
+            get
+            {
+                Debug.Assert(IsRunning);
+
+                return EvasObject.Geometry.Height;
+            }
+        }
+
+        public ElmFlutterView(EvasObject parent)
+        {
+            Parent = parent;
+        }
+
+        public ElmFlutterView(EvasObject parent, int initialWidth, int initialHeight)
+        {
+            Parent = parent;
+            InitialWidth = initialWidth;
+            InitialHeight = initialHeight;
+        }
+
+        /// <summary>
+        /// Starts running the view with the associated engine, creating if not set.
+        /// </summary>
+        public bool RunEngine()
+        {
+            if (Parent == null)
+            {
+                TizenLog.Error("The parent object is invalid.");
+                return false;
+            }
+
+            if (Engine == null)
+            {
+                Engine = new FlutterEngine();
+            }
+
+            if (!Engine.IsValid)
+            {
+                throw new Exception("Could not create a Flutter engine.");
+            }
+
+            var viewProperties = new FlutterDesktopViewProperties
+            {
+                width = InitialWidth,
+                height = InitialHeight,
+            };
+
+            View = FlutterDesktopViewCreateFromElmParent(ref viewProperties, Engine.Engine, Parent);
+            if (View.IsInvalid)
+            {
+                TizenLog.Error("Could not launch a Flutter view.");
+                return false;
+            }
+
+            EvasObject = new EvasObjectImpl(Parent, FlutterDesktopViewGetEvasObject(View));
+            if (!EvasObject.IsRealized)
+            {
+                TizenLog.Error("Could not get an Evas object.");
+                return false;
+            }
+
+            return true;
+        }
+
+        /// <summary>
+        /// Sets an engine associated with this view.
+        /// </summary>
+        public void SetEngine(FlutterEngine engine)
+        {
+            Engine = engine;
+        }
+
+        /// <summary>
+        /// Terminates the running view and the associated engine.
+        /// </summary>
+        public void Destroy()
```

Comment:
Is a separate Destroy() necessary? Can you handle it in ElmFlutterView's destructor?

Reply:
The .NET runtime's GC runs at an uncertain point of time during app execution so it didn't seem a good practice to use the destructor to release native resources.